pkg/moq/moq_test.go

```diff
@@ -2,15 +2,22 @@ package moq
 
 import (
 	"bytes"
+	"flag"
+	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"strings"
 	"testing"
 )
 
+// var update = flag.Bool("update", false, "update golden files")
```

Comment:
Remove this line, it is not necessary.

Reply:
This is just embarrassing 😅 - Fixed with amended commit.